### ISSUE TITLE
Revisiting DET/PRON for genitive (non-agreeing) pronominal forms

### ISSUE THREAD

**Issue by Nofenigma** [UniversalDependencies/docs#640]
It seems that the issue has not been discussed recently (though see #159, #269, and these guidelines: https://universaldependencies.org/v2/postags.html#revised-guidelines-for-pronominal-words), but some clarifications are still needed.

The problem is actual for many Indo-European languages and actually even beyond them as well.

In various European languages (Slavic, Romance, Germanic, Baltic, etc.), there are pronouns used in the possessive function which are not morphologically agreed with the modified nominals. Usually these are third person pronouns, but not only.

E.g., Swedish _min,_ _din_ have different gender and number forms, while _deras_ doesn't. In Russian, _moj,_ _tvoj_ behave as adjectives, but third person possessive pronouns are indeclinable.

Right now the situation is very messy. I have prepared a sample of examples from a number of European languages which illustrates this very well.

**Belarusian:** _iх_ — DET
**Czech:** _jejich_ – DET
**Finnish**: _heidän_ – PRON (nmod:poss) in PUD, but DET in FTB
**French:** _leur_ – DET (GSD), _leur_ – PRON (PUD) (and even _mon_ is PRON in French PUD); _leur_ actually has number forms (but not gender forms, as _mon_ or _ton),_ then it should be even a better candidate for DET probably?
**Latvian:** _viņu_ – PRON, but _det_
**Lithuanian:** _jų_ is PRON in Alksnis, DET in HSE
**Polish:** _ich_ is PRON, nmod
**Russian (SynTagRus)**: _их_ — DET
**Swedish** (all treebanks): _deras_ – PRON, nmod:poss

The situation is particularly interesting in the case of Finnish, as it seems that possessive pronouns are truly genitive, and in Estonian these are marked as follows: _minu_ ‘my’ is PRON and nmod.
The same should actually work for Lithuanian, as possessive pronouns are indeclinable there (unlike in Latvian), but still marked as DET in the HSE treebank - probably due to some proposals to choose the functionally-oriented annotation (as it has apparently happened in some Slavic treebanks), though I have not seen them properly.

I see two strategies beyond the decisions in particular treebanks, but still don't understand what is the current standard, especially considering the abovementioned instructions which are rather restrictive.

1. Functional equivalents of good candidates for determiners are marked as _det_ for the sake of uniformity.
E.g., that's why Russian _его,_ _eё,_ _их_ would get DET, though they are not agreed with the head

2. The principle of agreement is more crucial, then such indeclinable genitive pronouns remain PRON (with the relation nmod or probably even nmod:poss).

**Comment by dan-zeman** [comment]
It is true that Czech _jeho_ “his”, _její_ “her”, _jejich_ “their” do not inflect and do not show agreement with the modified (possessed) noun. It seems likely that these words have been derived from the genitive forms of the third person pronouns but only _jeho_ is ambiguous with a genitive form, while the other two are different: feminine genitive is _jí/ní_ “(of) her”, plural genitive is _jich/nich_ “(of) them”. So possessive pronouns are mostly different from genitive personal pronouns even in the third person.

The possessive pronouns are treated as determiners (`DET`). This tagging decision is based on the lexicon, not on the actual context (although context must be used to resolve the ambiguity between possessive _jeho_ – lemma _jeho_, and genitive _jeho_ – lemma _on_). The decision is quite uncontroversial for first-person and second-person possessives (_můj_ “my”, _tvůj_ “your”), which inflect like adjectives. Third-person possessives do not inflect, yet they are treated the same way because they belong to the same word category. One can view them as words with inflection syncretism.

Consequently, the `det` relation is used to attach possessive pronouns to the possessed nouns. It could be `det:poss` but the `:poss` subtype seems superfluous because they also have the feature `Poss=Yes`.

**Comment by Nofenigma** [comment]
Thanks for the clarifications about Czech, I was a little bit suspicious about this, but didn't check the paradigms. In any case, it seems that the criterion "if the possessive pronoun is undeclinable, but still formally different from the forms of personal pronouns, that is, it should get the DET label as a determiner" should be stated more explicitly.

> The decision is quite uncontroversial for first-person and second-person possessives (můj “my”, tvůj “your”), which inflect like adjectives. 

Why is it uncontroversial? The guidelines actually state that:
> Words that behave similar to adjectives are DET
and
>They are more likely to be used attributively (modifying a noun phrase) than substantively (replacing a noun phrase). 

Next, I would like to see an explanation for this:
> Third-person possessives do not inflect, yet they are treated the same way because they belong to the same word category.

How do we define that they belong to the same word category? Just because of their function? But if we have conjuncts, e.g., _Mary's and my house_, where _Mary's_ is the genitive form and _my_ is the possessive adjective, isn't the function the same here as well, though obviously these words are classified as different word categories?

In other words, so far the logic for choosing something as DET (vs. PRON) seems to be as follows:
- if a possessive word behaves as an adjective, it's likely to be DET;
- if a possessive word is non-adjectival (doesn't agree with its head), but has a form different from true personal pronouns, then it's likely to be DET;
- the residuals are likely to be classified as DET just because they have a similar function?

I also would like to emphasize that I'm mostly worried of the discrepancies in annotations of different treebanks, where the situations are really similar.

**Comment by dan-zeman** [comment]
>> The decision is quite uncontroversial for first-person and second-person possessives (můj “my”, tvůj “your”), which inflect like adjectives.

> Why is it uncontroversial? The guidelines actually state that:

>> Words that behave similar to adjectives are DET and They are more likely to be used attributively (modifying a noun phrase) than substantively (replacing a noun phrase).

It is uncontroversial because both the clues that you cite from the guidelines hold for it. These words behave like adjectives, are likely to be used attributively, and show gender and number agreement with the modified noun.

**Comment by dan-zeman** [comment]
>> Third-person possessives do not inflect, yet they are treated the same way because they belong to the same word category.

> How do we define that they belong to the same word category? Just because of their function? But if we have conjuncts, e.g., Mary's and my house, where Mary's is the genitive form and my is the possessive adjective, isn't the function the same here as well, though obviously these words are classified as different word categories?

Good question. In this particular case, the language-specific traditional grammar has the category "possessive pronouns", which contains the words _můj_ “my”, _tvůj_ “your”, _jeho_ “his”, _její_ “her” etc. The possessive semantics is one delimiting factor of the category. Another factor is adjective-like behavior: for the first and second person it is most visible in morphological agreement, but it is also visible (for all three persons) in word order preference: like adjectives, these words have a strong tendency to precede the modified noun, while genitive modifiers typically follow the modified noun. Finally, there is the pronominal factor, i.e., these words refer to the possessor simply by the `Person` feature. This makes them possessive pronouns (or determiners, in the UD taxonomy), and it distinguishes them from possessive adjectives such as _Mariin_ “Mary's” (don't get confused by the English translation, which converts the adjective into a genitive noun).

**Comment by dan-zeman** [comment]
> I also would like to emphasize that I'm mostly worried of the discrepancies in annotations of different treebanks, where the situations are really similar.

So am I. But it is difficult to come to a broadly acceptable compromise without leaving the impression that we are forcing language X to look like language Y, especially if the traditional grammars provide different views of the situation. Also, while the situation may be similar in the two languages, there may be a tiny difference that will weigh towards a different solution. And language X may have several neighbors, each of them similar to X, but each with differences pulling X in opposite directions.

That's why I tried to carefully explain the situation in Czech, but I'd prefer to wait with comments on the other languages you mentioned, because I know those languages less and I may be missing some important details.
